app/controllers/ticity_searches_controller.rb

```diff
@@ -1,2 +1,39 @@
 class TicitySearchesController < ApplicationController
+  
+  def index
+    ticity_searches = TicitySearch.all
+    render json: ticity_searches
+  end
+
+  def create
+    ticity_search = TicitySearch.create(ticity_search_params)
+    if ticity_search.valid?
+      render json: ticity_search
+    else
+      render json: ticity_search.errors, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    ticity_search = current_user.ticity_searches.find(params[:id])
+    ticity_search.update(ticity_search_params)
+    if ticity_search.valid?
+      render json: ticity_search
+    else
+      render json: ticity_search.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    ticity_search = current_user.ticity_searches.find(params[:id])
+    ticity_search.destroy()
+    render json: ticity_search
+  end
+
+  private
+
+  def ticity_search_params
+    params.require(:ticity_search).permit(:city1, :city2, :city3, :housing, :costofliving, :startups, :capital, :travel, :commute, :business, :safety, :healthcare, :education, :environment, :economy, :taxation, :internetaccess, :leisure, :tolerance, :outdoors, :user_id)
```

Comment:
Not a blocker: Being able to pass user_id here means that any logged in user could send whatever user_id they want and have a search get created for the passed user_id.

I suggest merging these params with {user_id: current_user.id} in a future PR.